chain/beacon/store.go

```diff
@@ -129,6 +132,14 @@ func (d *discrepancyStore) Put(b *chain.Beacon) error {
 	return nil
 }
 
+func (d *discrepancyStore) ForcePut(b *chain.Beacon) error {
```

Comment:
We would need a way to unwrap the store stack otherwise, because the Append Only store makes it illegal to Put if it's not after the Last stored beacon. Any idea how to disable the Append Store? 

And I don't think we can create a new boltdbstore store and write on the same file as the one used by the main process, that would be concurrent writes to the same file.

Reply:
Solved by keeping around the boltdbstore as an `InsecureStore` in the SyncManager